charts/redis-ha/values.yaml

```
@@ -169,6 +161,16 @@ haproxy:
     fsGroup: 1000
     runAsNonRoot: true
 
+  containerSecurityContext:
+    runAsNonRoot: true
+    allowPrivilegeEscalation: false
+    runAsUser: 1000
+    seccompProfile:
+      type: RuntimeDefault
+    capabilities:
+      drop:
+        - ALL
```

```suggestion
      - ALL
```

No indent sequences?